sale_product_pack/tests/test_sale_product_pack.py

```diff
@@ -8,8 +8,15 @@ class TestSaleProductPack(SavepointCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        main_company = cls.env.ref("base.main_company")
```

Comment:
Why does this always have to be associated with `base.main_company`? In that case, maybe ensure `self.env.company` is that main company...

Reply:
I agree